lib/tasks/revisions.rake

```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+
+desc 'Print debug information about post_revisions which cannot be deserialized'
+task 'revisions:debug_deserialization' => :environment do
+  puts "Checking #{PostRevision.count} PostRevision records in batches of 1000... this may take some time..."
+  sleep 1
```

Comment:
Why the `sleep`?

Reply:
Because otherwise it starts spamming the console with 1000s of lines before you get a chance to read the "this may take some time" message